assets/js/main.js

```diff
@@ -27,6 +27,14 @@ $("body").on('click', '.close-user--menu', function() {
   $('.user-menu').removeClass('open');
 });
 
+$(document).on('mouseup', function(e){
+	var menu = $(".user-menu");
+
+	if (!menu.is(e.target) && menu.has(e.target).length === 0){
+		menu.removeClass('open');
+	}
```

Comment:
If I am not wrong we were using spaces in place of tabs.

Reply:
I have the code editor set up in 2 spaces, but this would be a great opportunity to create a `.editorconfig` file to normalize how we all create code for the project. 👍